packages/zip-it-and-ship-it/src/runtimes/node/in_source_config/index.ts

```diff
@@ -176,38 +116,23 @@ export const parseSource = (source: string, { functionName }: FindISCDeclaration
       runtimeAPIVersion: 2,
     }
 
-    if (typeof configExport.schedule === 'string') {
-      result.schedule = configExport.schedule
-    }
-
-    if (typeof configExport.name === 'string') {
-      result.name = configExport.name
-    }
-
-    if (typeof configExport.generator === 'string') {
-      result.generator = configExport.generator
-    }
-
-    if (typeof configExport.timeout === 'number') {
-      result.timeout = configExport.timeout
-    }
-
-    if (configExport.method !== undefined) {
-      result.methods = normalizeMethods(configExport.method, functionName)
-    }
-
-    result.routes = getRoutes({
-      functionName,
-      methods: result.methods ?? [],
-      path: configExport.path,
-      preferStatic: configExport.preferStatic === true,
-    })
-
-    if (configExport.rateLimit !== undefined) {
-      result.trafficRules = getTrafficRulesConfig(configExport.rateLimit, functionName)
+    try {
+      const iscValues = isc.parse(configExport)
```

Comment:
😍

zod really is an incredibly nice piece of code.

Reply:
Yeah, I think it's going to make our future lives easier!